Source/UI/NewUI/BillingForm/View/BillingFormPhoneNumberText.swift

```diff
@@ -3,32 +3,41 @@ import PhoneNumberKit
 
 protocol BillingFormPhoneNumberTextDelegate {
     func updateCountryCode(code:Int)
+    func isInValidNumber()
```

Comment:
I think that `invalid` is a word. Whilst it has a prefix, it is a single word, so that method should probably be `isInvalidNumber`.

However, since it doesn't return anything, it probably should have a different name, something like `validateNumber`, which reads more like a function without output, which it appears to be

Reply:
This is a protocol. Would you apply the code suggestion directly and break everything else? It would be bizarre of me to suggest here and cause compilation errors if you were to approve my suggestion.

Reply:
Fixed